test/unit/helpers/admin/tabbed_nav_helper_test.rb

```diff
@@ -1,7 +1,8 @@
 require "test_helper"
 
 class TabbedNavHelperTest < ActionView::TestCase
-  test "#secondary_navigation_tabs_items for draft edition edit page" do
+  test "#secondary_navigation_tabs_items for draft edition edit page when user has editor access" do
```

Comment:
These were already like this, but the method being tested is `edition_nav_items`, not `secondary_navigation_tabs_items`.